base_user_role/models/__init__.py

```diff
@@ -1,3 +1,3 @@
 from . import role
 from . import user
-from . import res_groups
+from . import group
```

Comment:
By renaming the file to what it was in Odoo 13, you are moving *away* from the OCA convention. In this particular case, it also makes it more difficult to check what has changed in this file. Would you consider keeping the original name?

Reply:
You are right, I've applied the changes you are commenting.

Reply:
You have to change the name of the file also :S

Reply:
@MiquelRForgeFlow Thanks :smile: